src/main/java/com/github/zeger_tak/enversvalidationplugin/validate/AuditTableInformationMapValidator.java

```diff
@@ -57,4 +62,28 @@ public void validateAllExistingAuditTablesAreSpecified()
 			throw new ValidationException("The following audit tables are not configured in the audit table information map: " + auditTablesNotSpecified);
 		}
 	}
+
+	@Validate
+	public void validateAllContentTablesHaveAllColumnsInAuditTable() throws DataSetException, SQLException
+	{
+		final Set<String> missingAuditColumnNames = new HashSet<>();
+		for (AuditTableInformation auditTableInformation : auditTableInformationMap.values())
+		{
+			final Set<String> contentTableColumns = connectionProvider.getQueries().getAllColumns(auditTableInformation.getContentTableName());
+			final Set<String> auditTableColumns = connectionProvider.getQueries().getAllColumns(auditTableInformation.getAuditTableName());
+			final Set<String> notAuditedColumns = auditTableInformation.getColumnNamesPresentInContentTableButNotInAuditTable();
+
+			// @formatter:off
+			contentTableColumns.stream()
+				.filter(c -> !auditTableColumns.contains(c))
+				.filter(c -> !notAuditedColumns.contains(c.toUpperCase()))
+				.forEach(c -> missingAuditColumnNames.add(auditTableInformation.getAuditTableName() + "." + c));
+			// @formatter:on
+		}
+
+		if (!missingAuditColumnNames.isEmpty())
+		{
+			throw new ValidationException("The following columns are missing: " + StringUtils.join(missingAuditColumnNames, ", "));
```

Comment:
Persoonlijk zou ik de feedback aanpassen zodat er wordt gecommuniceerd dat de columns missen in de audit table.

Reply:
De feedback rapporteert TABLENAME.COLUMNAME, dus aan de table name zie je dat het over audit table kolommen gaat, ne ook welke audit table de kolom mist.